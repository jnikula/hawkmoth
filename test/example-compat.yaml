domain: c
directive-arguments:
  - example-compat.c
directive-options:
  transform: javadoc-liberal
example-title: Compat
example-priority: 95
