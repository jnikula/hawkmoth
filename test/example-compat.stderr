WARNING: example-compat.c:13: declaration of 'struct list' will not be visible outside of this function
WARNING: example-compat.c:13: declaration of 'enum mode' will not be visible outside of this function
